oat-sa/tao-core
# Feature/ADF-894/Bulk Import of item/test statistical metadata

# [ADF-894](https://oat-sa.atlassian.net/browse/ADF-894)

## How to test
1. Create any item/test properties
2. Create aliases for them
3. Create items/tests
4. Create CSV file in the following format:
    | itemId | testId | metadata_{alias1} | metadata_{alias2} | ... | metadata_{aliasN} |
    | --- | --- | --- | --- | --- | --- |
    | item 1 URI | OR Test 1 URI | Value 1 (can be empty) | Value 2 (can be empty) | ... | Value N (can be empty) |
    | item 2 URI | OR Test 2 URI | Value 1 (can be empty) | Value 2 (can be empty) | ... | Value N (can be empty) |

    Notes:
    * `itemId` - required if `testId` not provided
    * `testId` - required if `itemId` not provided
    * `alias1`, `alias2`, `aliasN` - necessary property aliases
    * delimiter - `;`
5. Open `Import statistical analysis metadata` page
6. Import CSV file
7. Check items/tests property values.

## Tasks
- [ ] [Add unit tests](https://oat-sa.atlassian.net/browse/ADF-922)
